fuzz/fuzz_targets/roundtrip.rs

```diff
@@ -0,0 +1,76 @@
+#![no_main]
+
+use png::FilterType;
+#[macro_use] extern crate libfuzzer_sys;
+extern crate png;
+
+fuzz_target!(|data: &[u8]| {
```

Comment:
It might be more straightforward to have this take a tuple of `(width, height, filter, compression, data)` rather than extracting those from the first bytes of a `&[u8]`.

Reply:
I would need to add `#[cfg_attr(fuzz), derive(Arbitrary)]` to these structs, and probably also a feature that enables the `arbitrary` dependency. If that's OK with you, I can do that.

Reply:
I was thinking of just making the argument type something like `(u16, u16, u8, u8, &[u8])` since IIRC the Arbitrary trait is already implemented for tuples.

Reply:
Done